arelle/plugin/validate/ESEF/Util.py

```diff
@@ -91,7 +91,7 @@ def checkImageContents(modelXbrl: ModelXbrl, imgElt: ModelObject, imgType: str,
             imageDoesNotMatchItsFileExtension = f"ESEF.{guidance}.imageDoesNotMatchItsFileExtension"
             incorrectMIMETypeSpecified = f"ESEF.{guidance}.incorrectMIMETypeSpecified"
             modelXbrl.error(imageDoesNotMatchItsFileExtension if isFile else incorrectMIMETypeSpecified,
-                _("Header type %(headerType)s has wrong image type: %(imgType)s"),
+                _("File type %(headerType)s inferred from file signature does not match the data URL mediatype (MIME type) %(imgType)s"),
```

Comment:
I don't think this is accurate.  `checkImageContents` is called [here](https://github.com/Arelle/Arelle/blob/553cae620ec20f6e10d042ae491dbbf9c25f02df/arelle/plugin/validate/ESEF/__init__.py#L449) with a non-`data:` URL.